ansible/systemd/infinite.service

```diff
@@ -9,7 +9,7 @@ Group=infinite
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=/bin/bash -c "docker-compose --project-directory /home/infinite/docker-files up --detach"
-ExecStop=/bin/bash -c "docker-compose --project-directory /home/infinite/docker-files stop"
+ExecStop=/bin/bash -c "docker-compose --project-directory /home/infinite/docker-files down"
```

Comment:
I'm confused by this file, does it run up before it runs down?

The process I followed when doing this manually was to:

1. docker-compose pull
2. docker-compose down
3. docker-compose up --detach

I assume the pull is happening somewhere else, or nothing would be working, and maybe I'm not understanding something about this file works, I do not spend a lot of time writing systemd service files

if we're calling down after we call up though, that definitely isn't right